conf/routes

```diff
@@ -28,6 +28,7 @@ GET     /api/vault/:vaultId/projectSummary/:projectId @controllers.FileListContr
 HEAD    /api/vault/:vaultId/:oid                    @controllers.VaultController.headTargetContent(vaultId, oid)
 GET     /api/vault/:vaultId/:oid/token              @controllers.VaultController.createSingleDownloadToken(vaultId, oid)
 GET     /api/rawdownload/:token                     @controllers.VaultController.singleTokenDownload(token)
+GET     /api/vaultDataJSONFile/:vaultId             @controllers.JSONDataFile.getAFile(vaultId)
```

Comment:
should be called something like `/api/duplicatereport/:vaultId/latest`

Reply:
Done.